src/Landing Page/Landing.js

```diff
@@ -27,14 +27,17 @@ export default function Landing() {
                   "Exploiting Machine Learning Solutions", "User Friendly", "Quantitative gains",
                   "Minimal Configuration Necessary", "No ML experience necessary"];
 
-  const setLoginModal = useLoginModalContext().setLoginModal;
+  
+  const {loginModal, setLoginModal} = useLoginModalContext();
   const auth = useAuth();
   const history = useHistory();
 
   function handleGetStartedOnClick() {
-    /* eslint-disable no-implicit-coercion, eqeqeq */
-    if (auth.user == null) setLoginModal(true);
-    else history.push("/console");
+    if (auth.user === "") {
+      return setLoginModal(true);
```

Comment:
We don't need to return anything here. Both setLoginModal and history.push return void anyway. Remove the `return`s